README.md

```diff
@@ -93,6 +93,8 @@ To use the default installation, you need to have [docker installed](https://doc
 
 **Windows users** must have Docker Desktop running.
 
+You system user also needs read/write access to the directory that contains your config file; typically this is also the directory where you are running the opsconsole.
```

Comment:
Do we also need/want to mention the $HOME directory caveat here for Linux superusers?

Reply:
It would be nice to add in. With the planned changes to error reporting from the provider, it should help expose that kind of error, so not essential to keep it here, but we probably should have it in some capacity. Let's leave it out for now